packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/file-uploader.js

```diff
@@ -58,7 +61,6 @@ async function uploadFile(s3Client, hostingBucketName, distributionDirPath, file
     Key: relativeFilePath,
     Body: fileStream,
     ContentType: contentType || 'text/plain',
-    ACL: 'public-read',
```

Comment:
File uploader is used in both the DEV and PROD setup, in the DEV setup, there is not CloudFront, and the S3 hosting bucket itself is private, if we don't set the ACL for the uploaded items, the contents won't be accessible. 
We need to distinguish between these two setup when trying to resolve this issue.

Reply:
I performed further updates on the repository and tested several options, add S3, turn into CloudFront, etc... The only thing that I am concerned now it the "Migrate" function. Can you give me some pointers if there are still issues.

Thanks